packages/ember-resolver/lib/initializers.js

```diff
@@ -4,7 +4,9 @@
   Ember.Application.initializer({
     name: 'container-debug-adapter',
 
-    initialize: function(container, app) {
+    initialize: function() {
+      var app = arguments[1] || arguments[0];
+      var container = arguments[1] ? arguments[0] : app;
```

Comment:
> container is not really necessary, app.register can be used below.

For which version(s) of ember does this statement apply though? Historically, the resolver has been fairly decoupled from specific framework versions -- it would be nice to keep it that way